### DISCUSSION TITLE
2022年终总结

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#307] [Announcements]
https://imcbc.cn/202212/2022-summary/?code=ca22b623abc800231a52

**Comment by penndu** [comment]
也时候写篇年终总结了！

**Comment by hqweay** [comment]
关注博主好一阵子了。感谢分享自己的经历，从中也看出了自己的影子（焦虑），嗐，不过博主比我对生活考虑得、也有执行力的多。希望咱们未来都能有点进步，一切都能好起来。

**Comment by panqiincs** [comment]
今年换了工作还换了城市，对你来说是「巨变」之年，想写的内容自然很多。从博客就能看出你的执行力真的好强，拿各种荣誉和奖项是水到渠成。祝在苏州和新的公司顺利！

**Comment by caibingcheng** [comment]
> @penndu
> 也时候写篇年终总结了！

（我盗了你的图👀，这里的图都是从去不广场盗过来的）

**Comment by caibingcheng** [comment]
> @hqweay
> 关注博主好一阵子了。感谢分享自己的经历，从中也看出了自己的影子（焦虑），嗐，不过博主比我对生活考虑得、也有执行力的多。希望咱们未来都能有点进步，一切都能好起来。

共勉，期望大家未来都能进步💪，都能如愿

**Comment by caibingcheng** [comment]
> @panqiincs
> 今年换了工作还换了城市，对你来说是「巨变」之年，想写的内容自然很多。从博客就能看出你的执行力真的好强，拿各种荣誉和奖项是水到渠成。祝在苏州和新的公司顺利！

谢谢！新工作和所想还是不太一样，不过先加油吧！

**Comment by coderiy** [comment]
最近看到好多博主都在开始写年终总结了hh

**Comment by caibingcheng** [comment]
> @JokerZhang66
> 最近看到好多博主都在开始写年终总结了hh

hh是的，快过年了嘛

**Comment by ryan4yin** [comment]
字里行间能感觉到 bbing 同学也确实有点书生气哈哈。
其实我也类似，写的很多文字也也老感觉自己脱离不了学生思维，但是我 2022 年这个问题好像好一些了，可能也是更油滑了吧 emmm

另外今年我的英语学习算是小有成就，9 月份到现在，4 个月涨了接近 2000 词，个人还挺满意的，也对之前定的明年 10000 词的目标更有底气了。
我从小到大，以前学英语也一直是三分钟热度，这一次为什么能坚持下来，我也还没得很想明白。

总之认识 bbing 同学也有大半年了，共同进步吧~

**Comment by ryan4yin** [comment]
哦还有，欢迎入坑 loveit 主题，我用了两年啦~

**Comment by caibingcheng** [comment]
> @ryan4yin
> 字里行间能感觉到 bbing 同学也确实有点书生气哈哈。
> 其实我也类似，写的很多文字也也老感觉自己脱离不了学生思维，但是我 2022 年这个问题好像好一些了，可能也是更油滑了吧 emmm
> 
> 另外今年我的英语学习算是小有成就，9 月份到现在，4 个月涨了接近 2000 词，个人还挺满意的，也对之前定的明年 10000 词的目标更有底气了。
> 我从小到大，以前学英语也一直是三分钟热度，这一次为什么能坚持下来，我也还没得很想明白。
> 
> 总之认识 bbing 同学也有大半年了，共同进步吧~

你能坚持下来太厉害了！向你学习💪！

**Comment by Macin20** [comment]
跟博主一样，我也是在武汉读的大学，当年差点儿也是进了华科，哈哈！**骚起来！**

**Comment by casteller** [comment]
向大佬学习

**Comment by windy177** [comment]
海阔凭鱼跃,天高任鸟飞。
我一直这么认为的，这个世界上的重重境遇，是外界和自己精神世界共同创建的。有点类似人体基因的表达，是环境和自身决定的。提到这里的意思是，不要复杂自己脑海中的社会，不要过度解读社会。用心去体验就好，勇敢表达自己就好。每一段经历都是可贵的，都是对你各个方面的历练。最重要、最可贵的是学会反思和思考。一个人在社会上流动的意义很大程度上是希望自己不会成为井地之蛙。跳出舒适的环境是好事，现在经历的不开心也是好事。都是你技术和性格成熟的必经之路。自己想改变 + 高执行力 + 对成功的渴望，必然会早就你的成功，时间问题罢了。路在脚下！共勉，朋友。

**Reply by caibingcheng** [reply]
感谢! 
"这个世界上的重重境遇，是外界和自己精神世界共同创建的", 对此有感, 外界环境是不容易变化的, 只有自己的精神世界是可以改变的, 所以当觉得社会很复杂, 离期望很远的时候, 更容易做的就是修心(反思, 思考, 执行和改变).
共勉!💪

**Comment by chz8bit** [comment]
感觉很棒啊，有改变的勇气！我也要学习。

**Comment by lewky** [comment]
很快就可以写2023年的总结了，我换了个公司后确实过得更舒服了，跟很多同事玩得很好，经常一起吃喝玩乐开黑打游戏，小公司有小公司的好处哈哈

**Comment by yunyit** [comment]
参考着这篇年终总结的小标题写好了2023年自己的，学习了很多，谢谢！祝新年快乐，新的一年一切顺利～～
